api/src/tests/converter_test.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{current_function_name, tests::new_test_context};
+use aptos_api_types::{AsConverter, MoveConverter};
+use aptos_vm::data_cache::AsMoveResolver;
+use move_core_types::{
+    account_address::AccountAddress,
+    resolver::MoveResolver,
+    value::{MoveStruct, MoveValue as VmMoveValue},
+};
+use serde::Serialize;
+use serde_json::json;
+use std::convert::TryInto;
+
+#[tokio::test]
+async fn test_value_conversion() {
```

Comment:
Because of stupidity. 😂

Reply:
hmm... in fact it has to be `tokio::test` because the test context etc assumes to be running in an async environment:

https://github.com/aptos-labs/aptos-core/blob/b1601d23fb157fee4c4932cd093d3809be8094a4/mempool/src/tests/mocks.rs#L77